extract_assets.py

```diff
@@ -250,7 +252,8 @@ def main():
                             "-c", ci_fmt,
                             "-p", tmp_pal.name
                         ])
-                    subprocess.run(cmd, check=True)
+                subprocess.run(cmd, check=True)
+                os.remove(name_file)
```

Comment:
This seems ok to me

Reply:
it's just windows who don't allow two program to open the same file